fh-messaging/lib/fhsrv.js

```diff
@@ -177,77 +254,13 @@ var msg = (function() {
 
 
   router.post('/:topic', function(req, res) {
```

Comment:
This is the API that is responsible for logging the messages into mongo so yes I think so. Multiple read requests could also be a problem for Mongo but I think that's a separate issue?